static/src/javascripts/projects/commercial/modules/header-bidding/a9/a9.js

```diff
@@ -34,19 +34,24 @@ const bidderTimeout: number = 1500;
 
 const initialise = (): void => {
     onIabConsentNotification(state => {
-        // typeof state === 'boolean' means CCPA mode is on
-        const canRun =
-            typeof state === 'boolean'
-                ? !state
-                : state[1] && state[2] && state[3] && state[4] && state[5];
-
-        if (!initialised && canRun) {
-            initialised = true;
-            window.apstag.init({
+        if (!initialised) {
+            const apstagConfig = {
                 pubID: config.get('page.a9PublisherId'),
                 adServer: 'googletag',
                 bidTimeout: bidderTimeout,
-            });
+            }
+            // typeof state === 'boolean' means CCPA mode is on
+            if (typeof state === 'boolean') {
+                initialised = true;
```

Comment:
Suggestions: put the `inititalised = true` lines outside the if to avoid repetition.

Reply:
had it outside initially, but would have implied that it's been initialised even when TCF consent is false, notice the else if condition.

Reply:
Absolutely right. Missed that.